File: dayFour/JavaScript/smallestToLargest.js
function smallestToLargestOf(numberList){
    newList = []
    let smallest = numberList[0]
    let largest = numberList[0]
    for(number = 0; number < numberList.length; number++){
            
        if (numberList[number] > largest)
        
            largest = numberList[number];
            
        if (numberList[number] < smallest)
        
            smallest = numberList[number];

    }
        
    let count = 0;
    for(number = smallest; number < largest+1; number++){
    
        newList[count] = number;
        count++;
    
    }

    return newList;
}
    
module.exports = {smallestToLargestOf};
    

File: dayFour/Python/smallest_to_largest.py
def smallest_to_largest_of(number_list):
    new_list = []
    smallest = number_list[0]
    largest = number_list[0]
    for count in range(len(number_list)):
        
        if number_list[count] > largest:
            
            largest = number_list[count]
            
        
        if number_list[count] < smallest:
            
            smallest = number_list[count]
  
            
    for new_number in range(smallest, largest+1):
    
        new_list.append(new_number)
      
            
    return new_list
    


